src/PBot.Tests.Integration/Helper.cs

```diff
@@ -5,27 +5,35 @@
 //copy pasted from Octokit.net
 public static class Helper
 {
-    static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
```

Comment:
In that case, I don't agree with removing it. If a field is designed to be readonly then we should embrace that and make it so.